tests/baselines/reference/genericCallAtYieldExpressionInGenericCall1.types

```diff
@@ -47,8 +47,8 @@ outer(function* <T>(value: T) {
 >                                                                             : ^^^^
 >outer : <A>(body: (value: A) => Generator<any, any, any>) => void
 >      : ^ ^^    ^^                                      ^^^^^    
->function* <T>(value: T) {  const result = yield* inner(value); // ok} : <T>(value: T) => Generator<never, void, never>
->                                                                      : ^ ^^     ^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+>function* <T>(value: T) {  const result = yield* inner(value); // ok} : <T>(value: T) => Generator<any, void, any>

```

Comment:
the new results here are better and match the other results that have `inner`~ function called beforehand (when its precomputed result is `yield*`ed)